src/configuration/index.ts

```diff
@@ -47,3 +47,22 @@ export const kovanConfiguration = (
     registrar: kovanRegistrar,
   };
 };
+
+export const rinkebyConfiguration = (
+  provider: ethers.providers.Web3Provider
+): Config => {
+  return {
+    subgraphUri: "https://api.thegraph.com/subgraphs/name/zer0-os/zns-rinkeby",
+    apiUri: "https://zns.api.zero.tech/api",
+    metricsUri: "https://zns-metrics-kovan.herokuapp.com", // todo
```

Comment:
remove todo maybe?

Reply:
It is a `kovan` URI still because we don't have a rinkeby metrics server setup yet, so leaving this until we do